src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1078,7 +1078,7 @@ format layout_optimizer::get_expected_format(quantize_node const& node) {
             else
                 return false;
         }
-
+        all_users_gemm &= (node.get_users().size() != 0);
```

Comment:
nit:
`all_users_gemm` can be initialized with that value

Reply:
fixed